stellar-contract-env-common/src/convert.rs

```diff
@@ -1,3 +1,7 @@
+// Copyright 2022 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
```

Comment:
Is this necessary? We don't do this in most of our repos.

Reply:
Opinions vary but not a lot -- eg. the apache project recommends putting it in every file, as do most lawyers I've ever asked. It clarifies 2 questions that can get lost otherwise: who has copyright and what license it came under.